packages/web-demo/src/app/bootstrap/components/background/background-demo-page/background-demo-page.component.html

```diff
@@ -11,16 +11,42 @@ <h2 class="bold">Infos</h2>
 </p>
 
 <h2 class="bold mt-huge">Default Bootstrap backgrounds</h2>
-<p class="my-4">
-  <label for="bg-opacity">Background opacity: {{bgOpacity}}</label>
-  <input id="bg-opacity" class="form-range" type="range" min="0" max="1" step="0.1" [(ngModel)]="bgOpacity">
-</p>
-<app-background-demo [bgOpacity]="bgOpacity" [colorArray]="['primary', 'success', 'danger', 'warning', 'info', 'light', 'gray', 'dark', 'white', 'transparent']"></app-background-demo>
+<div class="d-flex justify-content-center my-4">
+  <p class="form-check form-switch">
+    <input id="bg-opacity-1" class="form-check-input" type="checkbox" role="switch" [(ngModel)]="dropOpacity1">
+    <label class="form-check-label order-first" for="bg-opacity-1">Drop background opacity to 80%</label>
+  </p>
+</div>
+<div class="row">
+  <div class="col-12 col-lg-6">
+    <div class="p-3 bg-gray">
+      <app-background-demo [dropOpacity]="dropOpacity1" [colorArray]="['primary', 'success', 'danger', 'warning', 'info', 'light', 'gray', 'dark', 'white', 'transparent']"></app-background-demo>
+    </div>
+  </div>
+  <div class="col-12 col-lg-6">
+    <div class="p-3 bg-dark">
+      <app-background-demo [dropOpacity]="dropOpacity1" [colorArray]="['primary', 'success', 'danger', 'warning', 'info', 'light', 'gray', 'dark', 'white', 'transparent']"></app-background-demo>
+    </div>
+  </div>
+</div>
```

Comment:
Add the code examples here so it's clear how to apply the opacity.

Reply:
If I understand you correctly, we will apply the inline-style directly in the `background-demo-page.component.html` file.
But if we do this, the text inside the background-colored container will also be affected.

Is this in your interest?